dash_bio/component_factory/_clustergram.py

```diff
@@ -399,9 +276,13 @@ def __init__(
         self._width = width
 
         # convert line width to list if necessary
-        self._line_width = [0, 0]
```

Comment:
I removed this unnecessary line (maybe the intention was to initialize `self._line_width`?).